middlewares/hacker.middleware.js

```diff
@@ -523,6 +523,40 @@ async function updateHacker(req, res, next) {
     }
 }
 
+/**
+ * Updates a hacker that is specified by req.params.email, and then sets req.email
+ * to the email of the hacker, found in Account.
+ * @param {{params:{id: string}, body: *}} req
+ * @param {*} res
+ * @param {*} next
+ */
+async function updateHackerByEmail(req, res, next) {
+    const hacker = await Services.Hacker.updateOne(req.body.hacker._id, req.body);
```

Comment:
We’re updating the hacker in this method by ID, so why is it called “update hacker by Email”?